imap-proto/src/parser/rfc3501.rs

```diff
@@ -319,7 +319,7 @@ named!(opt_addresses<Option<Vec<Address>>>, alt!(
     ), |v| Some(v))
 ));
 
-named!(envelope<Envelope>, paren_delimited!(
+named!(pub (crate) envelope<Envelope>, paren_delimited!(
```

Comment:
Can we do `pub(crate)` here?

Reply:
Removing the whitespace correct? There are couple of other files with the space between `pub` and `(crate)`